incubating/composition-editor/lib/composition-editor.py

```diff
@@ -77,8 +83,13 @@ def main():
         # Rename Keys
         keyDict = { }
         oldKey, NewKey = payload.split(";")
```

Comment:
Got an error trying to run this with a single parameter in the `PAYLOAD` (ie `PAYLOAD='version'`)
```bash
Traceback (most recent call last):
  File "/composition-editor.py", line 114, in <module>
    main()
  File "/composition-editor.py", line 85, in main
    oldKey, NewKey = payload.split(";")
ValueError: not enough values to unpack (expected 2, got 1)
```

Something like this should help with that 
```python
splitKeys = payload.split(";") 
if len(keys) > 1: 
```

Reply:
Can you send me the ID of your build?  

This would happen in the old version of the step as it was only meant to handle key replacement not the removal.

Did you create the step in your personal account using a Docker image built from this repository, folder and branch?

Reply:
Ran it locally:
```bash 
$ git branch
  master
* sa-96-docker-composition-edit-key
$ docker build -t step-test .
...
$ docker run -it step-test sh
```
in the container:
```bash
$ wget https://raw.githubusercontent.com/codefresh-io/steps/master/incubating/composition-editor/example-composition.yaml
$ mkdir -p meta
$ DIRECTORY=/ YAMLFILE=example-composition.yaml EDIT_OBJECT=key PAYLOAD='version;ver' python3 /composition-editor.py
Set Environment Variable COMPOSITION_MODIFIED to true

$ DIRECTORY=/ YAMLFILE=example-composition.yaml EDIT_OBJECT=key PAYLOAD='services.db' python3 /composition-editor.py
Traceback (most recent call last):
  File "/composition-editor.py", line 114, in <module>
    main()
  File "/composition-editor.py", line 85, in main
    oldKey, NewKey = payload.split(";")
ValueError: not enough values to unpack (expected 2, got 1)
```

Reply:
User error, was not terminating my removal payload with a semicolon 😞